app/src/app/api/assignment/new/route.ts

```diff
@@ -0,0 +1,33 @@
+import type { NextApiRequest } from "next";
+import { prisma } from "@lib/prisma";
+import type { Score, ScoreDetail, Word } from "@/types";
+
+// 課題新規作成（ランダム）
+export async function POST(req: NextApiRequest) {
+	try {
+		const words = await prisma.word.findMany();
+		//ランダムでwordを取得
+		const word = words[Math.floor(Math.random() * words.length)];
+
+		// 課題を作成
+		const assignment = await prisma.assignment.create({
```

Comment:
```suggestion
		await prisma.assignment.create({
```
これって使わないならawaitからでいいのでは